backend/orchestration/recovery_orchestrator_enhanced.py

```
@@ -0,0 +1,1152 @@
+
+            # Verify recovery if enabled
+            if not is_dry_run and self.config.get("advanced.verify_recovery_success", False):
+                await self._verify_recovery(action)
+
+        except asyncio.TimeoutError:
+            action.success = False
+            action.error = f"Action timeout after {timeout}s"
+            action.result = RecoveryResult.FAILED
+            action.duration_seconds = time.time() - start_time
+
+            logger.error(f"❌ Recovery action timeout: {action_type}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)